yellowbrick/features/pcoords.py

```diff
@@ -239,26 +257,35 @@ def draw(self, X, y, **kwargs):
         # Convert from dataframe
         if is_dataframe(X):
             X = X.as_matrix()
+        if is_series(y):
+            y = y.as_matrix()
 
-        # Choose a subset of samples
-        # TODO: allow selection of a random subset of samples instead of head
+        # Get the total number of observations
```

Comment:
Good call.  Will do.

Reply:
@bbengfort - do you want this new ParallelCoordinates.fit() to return X and y?  Maybe only in some non-default verbose mode used for testing only?

Reply:
We do want to ensure the signatures for sklearn interface methods remain the same, for fit it is:

```python
class Estimator(object):
    
    def fit(self, X, y=None):
        return self     
```

This will ensure that helper methods such as `fit_transform` in pipelines continue to function as expected. 

To test, I recommend we mock `ParallelCoordinates.draw`, which `X` and `y` will be passed to, we can then inspect the output of `fit` there.